run_tests.sh

```diff
@@ -675,10 +675,11 @@ if [ "$test_script" = 'pytest' ]; then
         coverage_arg="--cov-report term --cov=lib"
     fi
     if [ -n "$marker" ]; then
-        "$test_script" -v --html "$report_file" $coverage_arg  $xunit_args $extra_args -m "$marker" "$@"
+        marker_arg="-m '$marker'"
     else
-        "$test_script" -v --html "$report_file" $coverage_arg  $xunit_args $extra_args "$@"
+        marker_arg=""
     fi
+    "$test_script" -v --html "$report_file" $coverage_arg $xunit_args $extra_args $marker_arg "$@"
```

Comment:
I think I've spent an hour yesterday trying everything I could think of to get `"not cwl_conformance"` as an argument. If you don't quote this you get ` -m ''\''not' 'cwl_conformance'\'''`, if you do you get `'-m '\''not cwl_conformance'\'''`.